### ISSUE TITLE
“Missing standard property 'transform' to go along with '-webkit-transform'.” when referring to a CSS property within @-webkit-keyframes

### ISSUE THREAD

**Issue by infinnie** [CSSLint/csslint#706]
```css
@-webkit-keyframes todo {
    0% {
        -webkit-transform: scaleZ(1);
        box-shadow: 0 0 4px rgba(0,0,0,.15);
        opacity: 0;
    }
    /* ... */
}
```

The problem is that if a browser supports `@-webkit-keyframes` it must support `-webkit-transform` and that should therefore neither be an error nor a warning.

**Comment by himanshuapril1** [comment]
I think it's always a best practice to have generic (without vendor prefix) property available below your vendor specific attributes, since usually vendor specific attributes are kept/considered only for compatibility fallback [till the time all the browsers are ready to upgrade them in order to support generic properties itself ;-) ].
